### ISSUE TITLE
Voiceover skips in-seat transfer list item in TripViewController

### ISSUE THREAD

**Issue by ualch9** [OneBusAway/onebusaway-ios#572]
From user feedback.

See below screenshots. The thick black outline is the focused voiceover element.
First row of the table is what Voiceover says.

# Expected output
| "Wall St & 5th Ave 11:23 AM" | "continues as 21 - Westwood VIllage Via 35th Ave SW" |
| --- | --- | 
| <img width="507" alt="Screen Shot 2021-11-19 at 11 55 10 AM" src="https://user-images.githubusercontent.com/22162410/142683800-f27e822d-371b-42be-964d-8eed43c0621f.png"> | <img width="507" alt="Screen Shot 2021-11-19 at 11 55 15 AM" src="https://user-images.githubusercontent.com/22162410/142683798-7c943793-6be9-4f80-a00b-2e0bf7f24308.png"> |

# Actual output
| "Wall St & 5th Ave 11:23 AM" | "Greenwood Ave. & N 85th St" |
| --- | --- | 
| <img width="507" alt="Screen Shot 2021-11-19 at 11 55 10 AM" src="https://user-images.githubusercontent.com/22162410/142683800-f27e822d-371b-42be-964d-8eed43c0621f.png"> | <img width="507" alt="Screen Shot 2021-11-19 at 11 55 15 AM" src="https://user-images.githubusercontent.com/22162410/142683798-7c943793-6be9-4f80-a00b-2e0bf7f24308.png"> |
